command/crypto/otp/verify.go

```diff
@@ -1,80 +1,75 @@
 package otp
 
 import (
-	"bufio"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"strings"
+	"time"
 
+	"github.com/pkg/errors"
 	"github.com/pquerna/otp"
 	"github.com/pquerna/otp/totp"
 	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/utils"
 	"github.com/urfave/cli"
 )
 
 func verifyCommand() cli.Command {
 	return cli.Command{
-		Name:        "verify",
-		Action:      cli.ActionFunc(verifyAction),
-		Usage:       "verify a one-time password",
-		UsageText:   `**step crypto otp verify**`,
+		Name:   "verify",
+		Action: cli.ActionFunc(verifyAction),
+		Usage:  "verify a one-time password",
+		UsageText: `**step crypto otp verify** <secret-file>
+[**--period**=<seconds>] [**--skew**=<num>] [**--length**=<size>]
+[**--alg**=<alg>] [*--time**=<time>]`,
 		Description: `**step crypto otp verify** does TOTP and HTOP`,
 		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:  "secret",
-				Usage: `A file containing the OTP secret`,
-			},
-			cli.IntFlag{
+			cli.UintFlag{
 				Name: "period",
 				Usage: `Number of seconds a TOTP hash is valid. Defaults to 30
 seconds.`,
 				Value: 30,
 			},
-			cli.IntFlag{
+			cli.UintFlag{
 				Name: "skew",
 				Usage: `Periods before or after current time to allow. Defaults
-to 0. Values greater than 1 require '--insecure'`,
+to 0. Values greater than 1 require '--insecure' flag.`,
 				Value: 0,
 			},
 			cli.IntFlag{
```

Comment:
Looks like because `otp.Digits()` expects int.

Reply:
I changed the others to Uint because that's what ValidateOptions expected. If you want I can change them back to Int and just cast to Uint like you did in generate.go. I figured this might be cleaner.